README.md

```diff
@@ -13,3 +17,16 @@ Options:
       --raw       Display the raw Markdown page without formatting
   -h, --help:     Display this help
 ```
+
+## Installation 
+
+```
```

Comment:
Would we like to add shell to that top line to signify it is commands to be run

Reply:
Sure 👍🏽